Mlem/Packages/Icons/Sources/Icons/Icon+Lemmy.swift

```diff
@@ -188,7 +188,7 @@ public extension Icon {
         
         @inlinable public var communityAvatar: Icon { community }
         public let instanceAvatar: Icon = .init("building.2.crop")
```

Comment:
We may want to apply the same logic here?

```suggestion
        public let instanceAvatar: Icon = .init("building.2.crop.circle")
```